### PR TITLE
Ccp 1794

### PR DESCRIPTION
**Pull request by Carsten89**
# Pull Request Template

## Description

Added PWA 6.7./6.8 support

## Type of change

- [x] Bug fix
- [ ] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

File: frontend/jest.config.js [hunk 1]
```diff
@@ -5,8 +5,10 @@ module.exports = {
   moduleNameMapper: {
     '^.+\\.(css|scss)$': 'identity-obj-proxy',
   },
+  transformIgnorePatterns: [
```
Comment (Carsten89) [inline frontend/jest.config.js]: Do we need this here?
Can't we simple only do 
`// Require the default configuration.
module.exports = require('@shopgate/pwa-unit-test/jest.config');`
in this file?

Comment (dqdquint) [reply frontend/jest.config.js]: Yup your right. I tested and updated.

File: frontend/.babelrc [hunk 2]
```diff
@@ -1,34 +1,72 @@
...
+      "@babel/plugin-syntax-dynamic-import",
+      {
+        "loose": true
+      }
+    ],
+    [
+      "module-resolver",
+      {
+        "cwd": "packagejson",
+        "alias": {
```
Comment (Carsten89) [inline frontend/.babelrc]: are these alias needed?

Comment (dqdquint) [reply frontend/.babelrc]: ah good catch. They are not needed. updated.

**Pull request merged**